ql/src/test/queries/clientpositive/materialized_view_parquet.q

```diff
@@ -189,8 +189,10 @@ alter materialized view mv1_parquet_n2 rebuild;
 alter materialized view mv1_parquet_n2 rebuild;
 
 explain cbo
-select name from emps_parquet_n3 group by name;
+select name, sum(empid) from emps_parquet_n3 group by name;
```

Comment:
Why are we changing existing queries? If it is useless then it's fine, if we want to test additional stuff then we should add new queries instead of modifying existing ones.

Reply:
The original test was actually hiding that projecting `ROW__ID.writeId` in case of insert-only tables always return `NULL`. However incremental MV rebuild was used and because of the `NULL` writeIds it did not inserted any records so the view data remained in an outdated state but the view was marked up to date by the system.
By adding the aggregation `sum(empid)` and the extra query after dropping the view the difference in the results showed up.

Reply:
Thanks for clarifying